types/plaid-link/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // Type definitions for non-npm package plaid-link-browser 2.0
-// Project: https://github.com/plaid/link
+// Project: https://github.com/plaid/react-plaid-link
 // Definitions by: Aaron Holderman <https://github.com/afholderman>
+//                 Brian Howald <https://github.com/bdhowald>
```

Comment:
Just to verify, are you wanting to become an owner of the package? If you're just looking for credit tracking, we do that via git history.

Reply:
I would like to become an owner, if the current owners are amenable to it.

Reply:
The other maintainers seem to be mostly inactive (hence, this appearing in the DT maintainer crew), so, you can be added.